velox/core/QueryConfig.h

```diff
@@ -277,6 +277,11 @@ class QueryConfig {
   static constexpr const char* kValidateOutputFromOperators =
       "debug.validate_output_from_operators";
 
+  /// If true, trade performance for memory. Optimizations including VectorPool
+  /// and Expr::evalWithMemo are disabled.
+  static constexpr const char* kOptimizeForLowMemory =
```

Comment:
Shall we just call it enable_expresssion_eval_cache?

Reply:
I think VectorPool is not limited to expression evaluation. It's part of ExecCtx inside OperatorCtx, so many operators can have VectorPool.